pkg/syncer/syncer.go

```diff
@@ -74,6 +75,7 @@ func StartSyncer(ctx context.Context, upstream, downstream *rest.Config, resourc
 	return nil
 }
 
+type mutatorGvrMap map[schema.GroupVersionResource]mutators.Mutator
```

Comment:
(No action required) If the intent is to only have a single mutator per gvr, maybe that suggests an `add` method that can throw an error if one is already defined?